treeherder/perf/migrations/0011_inc_extra_options_length.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-06 08:20
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('perf', '0010_fix_signature_uniqueness'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='performancesignature',
+            name='extra_options',
+            field=models.CharField(blank=True, max_length=832),
```

Comment:
832 characters seems a bit long. What's the reasoning for this number? :-)

Reply:
The old "extra_properties" field used to have a 1024 length. This is the closest value I could come to.
It allows us to have many options per alert, with a generous length for each option.

Reply:
If we reduce the limit on the number of extra options to 8, we can reduce this to 8*45=360.

Reply:
> If we reduce the limit on the number of extra options to 8, we can reduce this to 8*45=360.
Yep, that's still a reasonable limit.